main.cpp

```diff
@@ -1,15 +1,25 @@
 #include <fstream>
 #include <iostream>
+#include <cstring>
 
 #include <SDL2/SDL.h>
 
 #include "core/util.hpp"
+#include "core/cartridge.hpp"
+#include "core/mmu.hpp"
+#include "core/cpu.hpp"
+#include "core/ppu.hpp"
+#include "core/timer.hpp"
 
 const char TITLE[] = "gb-emulator";
 const int WIDTH = 160;
 const int HEIGHT = 144;
+const int NUM_PIXELS = 3 * 144 * 160;
```

Comment:
Fair point. This should be named `NUM_BYTES_OF_PIXELS`. This isn't only used for declaring the array, but also for the `memcpy` every frame, where I was trying to avoid an unnecessary calculation every frame.